src/Server.py

```diff
@@ -15,6 +16,10 @@
 from . import Database
 from src.Errors import ReturnResultValueLessThanOne
 
+# type alias
+Word_list = List[str]
+Word_lists = List[Word_list]
+
```

Comment:
Correct me if I am wrong, So this is basically creating a new Type  called `Word_lists` which is a `List` of Type `Word_list`, which is in turn a `List of str` .. So this is a recursive type

Reply:
You are right, just a fancy new Python feature makes type more expressive

Reply:
LGTM